ansible/host_vars/localhost

```diff
@@ -0,0 +1,16 @@
+---
+# FIXME(w-miller): set this more cleverly?
+base_path: "~/tenks/"
+allocations_file_path: "{{ base_path + '/allocations.yml' }}"
+
+vm_types:
+  type0:
+    memory_mb: 4096
+    vcpus: 4
+    volumes:
+      - capacity: 10GB
+    physical_networks:
+      - physnet1
+
+specs:
+  type0: 4
```

Comment:
These defaults could easily overload a machine. I'd either make them more conservative (e.g. 1 small node), or empty, with a good comment explaining their format.

Reply:
The default here was really just left to make my testing easier - I think this should be empty as you say.